src/components/StorageOverview/DataResiliency/DataResiliency.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, UtilizationBar } from 'patternfly-react';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../Dashboard/DashboardCard';
+import { StorageOverviewContext } from '../StorageOverviewContext';
+import { InlineLoading } from '../../Loading';
+import { getDataResiliencyData } from '../../../selectors/prometheus/selectors';
+
+const DataResiliencyStatusBody = ({ status }) =>
+  status ? (
+    <React.Fragment>
```

Comment:
nitpick: the rest of web-ui-components code has `Fragment` in import explicitly.

Reply:
I would say that we use `React.Fragment` more frequently than `Fragment` import :)

Reply:
done !! yeah @rawagner frequently!!